docs/DeveloperGuide.md

```diff
@@ -51,7 +53,8 @@ For example, the `Logic` component (see the class diagram given below) defines i
 
 **How the architecture components interact with each other**
 
-The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The *Sequence Diagram* below shows how the components interact with each other for the scenario where the
+ user issues the command `delete supplier 12`.
```

Comment:
The delete command here should be changed as well, similar to that pointed by Zhenlin is another section below :)